res/openvpn-gui-res-pl.rc

```diff
@@ -531,18 +532,18 @@ Prawdopodobnie OpenVPN nie jest zainstalowany."
 z prawami administratora aby uaktualnić rejestr."
     IDS_ERR_READ_SET_KEY "Błąd odczytu/zmiany klucza rejestru ""%ls""."
     IDS_ERR_WRITE_REGVALUE "Błąd zapisu rejestru ""HKEY_CURRENT_USER\\%ls\\%ls""."
-    IDS_ERR_PLAP_REG "Failed to enable Start Before Logon (error = %d)"
-    IDS_ERR_PLAP_UNREG "Failed to disable Start Before Logon (error = %d)"
+    IDS_ERR_PLAP_REG "Błąd podczas włączania łączenia przed logowaniem (błąd = %d)"
```

Comment:
IDS_ERR_PLAP_REG "Błąd podczas nawiązywania połączenia przed logowaniem (błąd = %d)"

Reply:
I know it sounds strange, but it actually describes what happens. It does not refer to “connection error”, but to “enabling certain type of connections”.